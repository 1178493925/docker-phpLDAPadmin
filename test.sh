#!/bin/sh

# Usage
#Â sudo ./test.sh 
# add -v for verbose mode (or type whatever you like !) :p

. test/config
. test/tools/run.sh

run_test tools/build-container.sh "Successfully built"
run_test simple.sh "Use the menu to the left to navigate"
run_test link.sh "Logged in as:"

. test/tools/end.sh
