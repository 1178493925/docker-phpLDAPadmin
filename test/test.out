docker run --name phpldapadmin-test-container  -d phpldapadmin-test 
